test/test.py

```diff
@@ -143,18 +144,17 @@ def _canned_bundle(self, directory, uuid, version):
         else:
             return None, None
 
-    def _load_bundle(self, uuid, version, replica='aws', deployment=None):
+    def _load_bundle(self, uuid, version, replica='aws', deployment='prod'):
```

Comment:
```suggestion
   
    prod = 'prod'

    def _load_bundle(self, uuid, version, replica='aws', deployment=prod):
```

Reply:
I am unsure of the benefit of this suggested change.